plugins/automox/icon_automox/actions/update_device/action.py

```diff
@@ -27,9 +28,9 @@ def run(self, params={}):
         payload = {
             "server_group_id": server_group_id,
             "ip_addrs": current_device_details["ip_addrs"],
-            "exception": params.get(Input.EXCEPTION, current_device_details["exception"]),
-            "tags": params.get(Input.TAGS, current_device_details["tags"]),
-            "custom_name": params.get(Input.CUSTOM_NAME, current_device_details["custom_name"]),
+            "exception": params.get(Input.EXCEPTION, current_device_details.get("exception", False)),
+            "tags": params.get(Input.TAGS, current_device_details.get("tags", [])),
+            "custom_name": params.get(Input.CUSTOM_NAME, current_device_details.get("custom_name", "")),
         }
```

Comment:
Just wondering if we should use `helper.clean` on that payload to remove empty fields.

Reply:
If not provided, a default value needs to be set for this request to be valid, so we can't remove them or null them.